pallets/rmrk-core/src/functions.rs

```diff
@@ -251,18 +251,21 @@ where
 		collection_id: CollectionId,
 		nft_id: NftId,
 		new_owner: AccountIdOrCollectionNftTuple<T::AccountId>,
-	) -> sp_std::result::Result<T::AccountId, DispatchError> {
+	) -> sp_std::result::Result<(T::AccountId, bool), DispatchError> {
 		// Get current owner for child removal later
 		let parent = pallet_uniques::Pallet::<T>::owner(collection_id, nft_id);
 		// Check if parent returns None which indicates the NFT is not available
-		ensure!(parent.is_some(), Error::<T>::NoAvailableNftId);
+		ensure!(parent.is_some(), Error::<T>::NoAvailableNftId); // <- is this error wrong?
```

Comment:
This would be a case where an NFT is not valid bc there is no owner for the NFT which would mean it doesn't exist or is a burned NFT. Is there a case where a NFT could not have an owner?

Reply:
maybe i was reading it wrong (sounds like it) but i thought the check was for ownership, but the error was for existence.  sounds like the comment can be removed.